contracts/core/OriginCore.sol

```diff
@@ -412,4 +474,120 @@ contract OriginCore is OriginCoreInterface, OriginCoreConfig {
 
         return kernelHash;
     }
+    /**
+     * @notice private method to check vote validity.
```

Comment:
The notice should describe what the function does and not necessarily whether it is `private`.
Please start the sentence with uppercase.
(See also other functions' NatSpec)